EmbeddedPkg/Library/PrePiLib/FwVol.c

```diff
@@ -886,14 +888,19 @@ FfsProcessFvFile (
   //
   BuildFvHob ((EFI_PHYSICAL_ADDRESS)(UINTN)FvImageInfo.FvStart, FvImageInfo.FvSize);
 
+  // MU_CHANGE Start
+  Status = FfsGetVolumeInfo (ParentVolumeHandle, &ParentVolumeInfo);
+  ASSERT_EFI_ERROR (Status);

```

Comment:
Not sure if this gets fixed by a CodeQL update, but since we are adding this here, graceful exit instead of assert?

Reply:
It won't get fixed by existing CodeQL changes, but I left a note here for the Mu Tiano Plus CodeQL work [[Feature]: CodeQL Fixes](https://github.com/microsoft/mu_tiano_plus/issues/290).